drivers/oled/oled_driver.c

```diff
@@ -19,6 +19,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #    include "spi_master.h"
 #elif defined(OLED_TRANSPORT_I2C)
 #    include "i2c_master.h"
+#    include "keyboard.h"
```

Comment:
```suggestion
#    if defined(USE_I2C) && defined(SPLIT_KEYBOARD)
#        include "keyboard.h"
#    endif
```

Reply:
Include should go down to line 24.

Reply:
You mean outside of the ifdef block for `OLED_TRANSPORT_*`? 

If so, `is_keyboard_master` is only called if the transport is i2c, `USE_I2C` is defined, and split keyboard is enabled. For reference:
https://github.com/qmk/qmk_firmware/blob/422d502903c43c19007c95e6206d34f22da63092/drivers/oled/oled_driver.c#L289-L293

Reply:
> You mean outside of the ifdef block for OLED_TRANSPORT_*?

Correct. It won't hurt anything to have it unconditionally included. Actually I'm not sure why it's only needed for I2C...

Reply:
> > You mean outside of the ifdef block for OLED_TRANSPORT_*?
> 
> Correct. It won't hurt anything to have it unconditionally included. Actually I'm not sure why it's only needed for I2C...

Looking at the code, I think maybe line **290** was added to make sure that the slave **MCU** not to do the "rotation" part because I noticed it will just return true assuming the master **MCU** already did it. This is why I think you can only run one **Oled** (maybe two but showing the same thing). 

Where if a split is using serial I don't think the **Oled** "rotation" matters because maybe each half can control it's **Oled**

Reply:
Ah yeah, for split boards using I2C as the transport, only one OLED is supported, because the slave half cannot be both slave to the master half and master to an OLED at the same time.

Reply:
Yeah, the original code check was added for splits that use i2c, so that they could run an oled.  It would only need to be run on the master side, since that is what will get all the data/info/etc, anyways. 

As for the includes themselves, I'm not particular in how, it just feels cleaner to have them in the initial block, rather than separate.  But that's absolutely personal preference.